packages/default/scss/window/_layout.scss

```diff
@@ -38,9 +39,9 @@
 
     // Title bar title text
     .k-window-title {
-        margin: 0;
+        margin: (-$window-title-lineheight / 8) 0;
```

Comment:
This evaluates to `0.1625em` in the default theme. Why? Is this perhaps connected to the `$window-titlebar-padding-y`?

Reply:
We have updated the `$window-title-lineheight` from 1 to 1.3. We have to add negative margins at top and bottom close to the updated higher line-height style (0.3)